packages/api/src/routes/medical/fhir-r4-proxy.ts

```diff
@@ -5,6 +5,24 @@ import { Config } from "../../shared/config";
 import { Util } from "../../shared/util";
 import { asyncHandler, getCxIdOrFail } from "../util";
 
+export type fhirLink = {
+  relation: "self" | "next" | "previous";
+  url: string;
+};
+
+export type fhirEntry = {
+  fullUrl: string;
+  resource: {
+    // TODO: Incomplete type
+    resourceType: string;
+    id: string;
+  };
+  search: {
+    // TODO: Incomplete type
+    mode: string;
+  };
+};
```

Comment:
Likely can use `@medplum/fhirtypes`?

Reply:
https://www.medplum.com/docs/search/paginated-search

helpful reference. The link type in Medplum is BundleLink within the Bundle type